src/localisation/string_ids.h

```diff
@@ -114,6 +114,7 @@ enum {
 	STR_CANT_RAISE_LAND_HERE = 880,
 
 	STR_LOAD_GAME = 882,
+	STR_SAVE_GAME_AS = 5512,
```

Comment:
I remember @IntelOrca saying so and having sorted the list himself a couple of times.